docs/src/datamodel/index.rst

```diff
@@ -0,0 +1,385 @@
+The FMU results data model
+##########################
+
+This section describes the data model used for FMU results when exporting with
+fmu-dataio. For the time being, the data model is hosted as part of fmu-dataio.
+
+The data model described herein is new and shiny, and experimental in many aspects.
+Any feedback on this is greatly appreciated. The most effective feedback is to apply
+the data model, then use the resulting metadata.
+
+The FMU data model is described using a `Pydantic <https://pydantic.dev/>`__ model
+which programmatically generates a `JSON Schema <https://json-schema.org/>`__. 
```

Comment:
This text is mostly a copy of the previous datamodel.rst, with small changes like this